packages/chart/src/scss/main.scss

```diff
@@ -153,23 +153,46 @@
   .legend-container {
     background: #fff;
     width: 100%;
-    padding: 15px;
     vertical-align: top;
     text-align: left;
     border: 1px solid var(--lightGray);
     position: relative;
 
+    h3,
+    p,
+    tspan,
+    div {
+      // TODO: when we have updated each section to use the new font, we can transition to using a global style
```

Comment:
Would you rather keep this here or transition it into core? .chart-container { ...styles }. I'll let you make that call, but if we're working across different packages you might consider moving it so other developers see it across the different packages.

Reply:
great idea